### DISCUSSION TITLE
Sample .imh and .pix files

### DISCUSSION THREAD

**Discussion by cruzzil** [iraf-community/iraf#341] [General]
Can anyone provide a sample .imh and .pix file so I can test the IRAF->FITS conversion code in cfitsio library?

**Comment by olebole** [comment]
`dev$pix` is the standard sample file. Alternatively, one could use the [rfits](https://iraf.readthedocs.io/en/latest/tasks/dataio/rfits.html) task to convert from FITS to IRAF. The [mkexamples](https://iraf.readthedocs.io/en/latest/tasks/noao/artdata/mkexamples.html) task can be used to create example images as well.

**Reply by cruzzil** [reply]
Thanks for the prompt response!
